packages/node_modules/@webex/plugin-messages/src/messages.js

```diff
@@ -388,6 +388,9 @@ const Messages = WebexPlugin.extend({
             }
           }
         }
+        if ((typeof activity.parent === 'object') && (activity.parent.id)) {
```

Comment:
```suggestion
        if (activity.parent && activity.parent.id) {
```
should work as well and is a bit more consistent with the rest of the sdk, but not a blocker.

Reply:
Done, modified some of the other conditions in this function to match that style as well.